### PR TITLE
Intel grpc dl streamer module

### PR DESCRIPTION
**Pull request by fvneerden**

### CODE REVIEW

**Comment by fvneerden** [comment]
#sign off

File: MediaGraph/topologies/grpcExtensionOpenVINO/2.0/readme.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Analyzing live video using gRPC Extension to send images to the OpenVINO(TM) DL Streamer - Edge AI Extension module from Intel
+
+This topology enables you to run video analytics on a live feed from an RTSP-capable camera. The gRPC Extension allows you to create images at video frame rate from the camera are converted to images, and sent to the [ OpenVINO(TM) DL Streamer - Edge AI Extension module from Intel](https://aka.ms/lva-intel-ovms) module provided by Intel. The results are then published to the IoT Edge Hub. You can see how this topology is used in [this](https://aka.ms/lva-intel-grpc) tutorial.
```
Comment (anilmur) [inline MediaGraph/topologies/grpcExtensionOpenVINO/2.0/readme.md:3]: Too much Intel: drop the "provided by Intel" in this sentence

The gRPC Extension allows you to create images at video frame rate from the camera are converted to images, and sent to the [OpenVINO(TM) DL Streamer - Edge AI Extension module from Intel](https://aka.ms/lva-intel-ovms) module provided by Intel.

**Review by anilmur — APPROVED** [review]
Approved, but one edit requested

**Comment by fvneerden** [comment]
follow up in https://github.com/Azure/live-video-analytics/pull/188

**Pull request closed without merging**